test/cases/020_kernel/211_tags_5.4.x/test.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+# SUMMARY: Test build and insertion of kernel modules
+# LABELS:
+# REPEAT:
+
+set -e
+
+# Source libraries. Uncomment if needed/defined
+#. "${RT_LIB}"
+. "${RT_PROJECT_ROOT}/_lib/lib.sh"
+
+NAME=tags
+
+clean_up() {
+	docker rm ${ctrid}
+	/bin/rm -f ${BUILDERFILE}
+}
+trap clean_up EXIT
+
+# check the kernel images for tags, labels, files
+KERNEL=linuxkit/kernel:5.4.116
+BUILDER=${KERNEL}-builder
+BUILDERFILE=/tmp/kernel-builder-${KERNEL}
```

Comment:
I would do this in the local directory and add a `.gitignore`, but `/tmp` works too